### PR TITLE
[WIP] Prepare initial third party services

### PR DESCRIPTION
**Pull request by Fryguy**
Prepare initial third party services

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
-[![Test Coverage](https://codeclimate.com/github/ManageIQ/manageiq-providers-ibm_cloud/badges/coverage.svg)](https://codeclimate.com/github/ManageIQ/manageiq-providers-ibm_cloud/coverage)
+[![Build Status](https://travis-ci.com/ManageIQ/manageiq-providers-ibm_cloud.svg?branch=master)](https://travis-ci.com/ManageIQ/manageiq-providers-ibm_cloud))
+[![Maintainability](https://api.codeclimate.com/v1/badges/41e71ad240a79b0be9d9/maintainability)](https://codeclimate.com/github/ManageIQ/manageiq-providers-ibm_cloud/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/41e71ad240a79b0be9d9/test_coverage)](https://codeclimate.com/github/ManageIQ/manageiq-providers-ibm_cloud/test_coverage)
 [![Security](https://hakiri.io/github/ManageIQ/manageiq-providers-ibm_cloud/master.svg)](https://hakiri.io/github/ManageIQ/manageiq-providers-ibm_cloud/master)
 
-[![Chat](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ManageIQ/manageiq-providers-ibm_cloud?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (chessbyte) [inline README.md:9]: why not create a Gitter room for this repo?

Comment (agrare) [inline README.md:9]: I can create a gitter room if we want to keep the Chat badge

Comment (Fryguy) [reply README.md:9]: This is generated automated at the moment but there is always some manual changes I have to make...this is one of them (hence why it's WIP)

File: .codeclimate.yml [hunk 2]
```diff
@@ -28,9 +28,3 @@ plugins:
   rubocop:
     enabled: true
     config: ".rubocop_cc.yml"
```
Comment (agrare) [inline .codeclimate.yml:30]: You'll need `channel: rubocop-0-69` here to fix the codeclimate failure but we can do that in a follow-up PR

**Pull request merged**